Demo/UI Tests/ThreeDSecure UI Tests/ThreeDSecure_V1_UITests.swift

```diff
@@ -16,14 +17,22 @@ class ThreeDSecure_V1_UITests: XCTestCase {
         app.launchArguments.append("-ClientToken")
         app.launchArguments.append("-Integration:BraintreeDemoThreeDSecurePaymentFlowViewController")
         app.launch()
+        
+        springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
 
         _ = app.cardNumberTextField.waitForExistence(timeout: 10)
     }
+    
+    private func waitForAuthDialogAndTapButton(named buttonName: String) {
+        _ = springboard.buttons[buttonName].waitForExistence(timeout: .threeDSecureTimeout)
+        springboard.buttons[buttonName].tap()
+    }
```

Comment:
I wonder if this will be more stable that our current implementation 🤞

Reply:
Yeah it's hard to tell the tests where definitely 🧯 🔥  for a while haha. They're passing locally but it looks like a few are 🔴  in CI. I think the 20 second timeout might be a goto value.